src/Farmer/Arm/DocumentDb.fs

```diff
@@ -36,6 +38,10 @@ module DatabaseAccounts =
                                   partitionKey =
                                    {| paths = this.PartitionKey.Paths
                                       kind = string this.PartitionKey.Kind |}
+                                  uniqueKeyPolicy = 
+                                   {| uniqueKeys = 
+                                      this.UniqueKeyPolicy.UniqueKeys 
+                                      |> List.map (fun k -> {| paths = k.Paths |}) |}
```

Comment:
Can this be simplified to just `uniqueKeyPolicy = this.UniqueKeyPolicy`?

Reply:
I was just manually mapping through the `this.UniqueKeyPolicy.UniqueKeys` to manually create camel cased versions of the children in the object graph (since there is an extra layer beneath that).  I agree that it's a little cumbersome compared to just using a camel case serialization in Newtonsoft.Json, but I was trying to keep it simple and just copy the methodology that was already in-place above.

Btw, here is the desired output:

![image](https://user-images.githubusercontent.com/1030435/82548235-fd5dc700-9b28-11ea-8c18-be3d029907c5.png)

Reply:
And for good measure, here is what was generated if two unique keys are entered via Azure Portal (and what this currently generates as well):

![image](https://user-images.githubusercontent.com/1030435/82549007-3f3b3d00-9b2a-11ea-9c7e-8812abecd130.png)